src/math/Matrix3.js

```diff
@@ -114,6 +114,62 @@ Object.assign( Matrix3.prototype, {
 
 	}(),
 
+	makeBasis: function ( xAxis, yAxis, zAxis ) {
+
+		this.set(
+			xAxis.x, yAxis.x, zAxis.x,
+			xAxis.y, yAxis.y, zAxis.y,
+			xAxis.z, yAxis.z, zAxis.z
+		);
+
+		return this;
+
+	},
+
+	lookAt: function () {
+
+		var localRight = new Vector3();
+		var worldRight = new Vector3();
+		var worldUp = new Vector3( 0, 1, 0 );
```

Comment:
> This is fixing world up to be +y ?

`worldUp` could be actually configurable. I've hard-wired this since `three.js` assumes Y-Up.

Reply:
some awkward people (ie me*) work with up = +z,  the existing Matrix4.lookAt() has an
exception around for the z up case #11543 already.

* geographic models fit better.

Reply:
If we find a way to make it configurable, it should be no problem. Do you have any ideas?

Reply:
Damn. I realize right now that this PR breaks all user code where `camera.up.set( 0, 0, 1 )` is already set. https://threejs.org/examples/webgl_loader_pcd.html is such a problematic case. You would now have to change `worldUp` which is not a backward-compatible approach. Too bad, it seems we can't go this way.

Reply:
Do you mean that these use cases will break unless the user also does `camera.forward.set( 0, -1, 0 )`?

Reply:
I don't know why yet but this does not solve the issue. It works if you remove `camera.up.set( 0, 0, 1 )` and set `worldUp` to `(0, 0, 1)`. Even then, for some reasons `TransformControls` seems to have problems with this approach (it internally uses `Matrix4.lookAt()`)...

Reply:
After a closer look, the problem is related since the new `Matrix3.lookAt()` interprets `Object3D.up` differently than `Matrix4.lookAt()`. That's the reason why both methods do not work together like in the example. From my point of view, `Matrix3.lookAt()` is the better implementation since it's a more clear implementation of what's actually happening in the method: A given `Object3D.up` vector is mapped like all other basis vectors (forward, right) to the counterpart of the world coordinate frame. This is the way `.lookAt()` should work and `Matrix3.lookAt()` is exactly implemented like that. 

Even if we introduce `THREE.WorldUp` to allow a configuration (e.g. to enable Z-UP) and then use this value in `Matrix3.lookAt()`, we would have to migrate all code that uses `Matrix4.lookAt()`  to `Matrix3.lookAt()`. So introducing this new approach is a breaking change.

Reply:
Thanks for investigating!